src/clndr.js

```diff
@@ -874,9 +874,8 @@
                 self.options.selectedDate =
                     self.getTargetDateString(event.currentTarget);
                 // Handle "selected" class
-                $currentTarget
-                    .siblings().removeClass(classes.selected).end()
-                    .addClass(classes.selected);
+                $container.find('.' + classes.selected).removeClass(classes.selected);
+                $currentTarget.addClass(classes.selected);
```

Comment:
Thanks @pkostic, could you explain the difference / need to use container.find vs. the siblings call?